portfolio/src/main/webapp/photo.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="node_modules/bulma/css/bulma.min.css" />
+        <link rel="stylesheet" href="style.css" />
+        <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
+        <script src="node_modules/jquery/dist/jquery.min.js"></script>
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <section class="hero is-light is-large">
+            <div class="hero-head">
+                <nav class="navbar">
+                    <div class="container">
+                        <div class="navbar-brand">
+                            <a class="navbar-item">
+                                <h1>Iris Liu</h1>
+                            </a>
+                        </div>
+                        <div id="navbarMenuHeroB" class="navbar-menu">
+                            <div class="navbar-end">
+                                <a class="navbar-item" href="index.html">Home</a>
+                                <a class="navbar-item" href="me.html">About Me</a>
+                                <a class="navbar-item is-active" href="photo.html">Photography</a>
+                                <span class="navbar-item">
+                                    <a class="button is-inverted navbar-icon-background" href="https://linkedin.com/in/jiayi-liu-1010" target="_blank">
+                                        <span class="icon has-text-dark is-large">
+                                            <i class="fab fa-linkedin"></i>
+                                        </span>
+                                    </a>
+                                </span>
+                                <span class="navbar-item">
+                                    <a class="button is-inverted navbar-icon-background" href="https://github.com/irisliu77" target="_blank">
+                                        <span class="icon has-text-dark is-large">
+                                            <i class="fab fa-github-square"></i>
+                                        </span>
+                                    </a>
+                                </span>
+                            </div>
+                        </div>
+                    </div>
+                </nav>
+            </div>
+            <div class="hero-body">
+                <div class="columns is-centered">
+                    <div class="column is-one-third">
+                        <div class="field">
+                            <div class="control">
+                                <input class="input" id="photo-input" type="text" placeholder="Try portrait, cityview, arts...">
+                                <button class="button" id="filter-button">Filter</button>
+                                <button class="button" id="unfilter-button">Unfilter</button>
```

Comment:
The unfilter text is too big for its container - it's extending outside of the button, just FYI.

Reply:
solved